.github/workflows/ci.yml

```diff
@@ -139,22 +139,21 @@ jobs:
         .github/workflows/tests.sh
 
     - name: Upload coverage report
-      if: matrix.python-version == 3.5
-      uses: codecov/codecov-action@v1.0.5
+      if: matrix.python-version == 3.5 && github.repository == 'aiidateam/aiida-core'
+      uses: codecov/codecov-action@v1
       with:
-        token: ${{ secrets.CODECOV_TOKEN }}
         name: aiida-pytests-py3.5-${{ matrix.backend }}
         flags: ${{ matrix.backend }}
         file: ./coverage.xml
-        fail_ci_if_error: false
+        fail_ci_if_error: true
```

Comment:
I guess this is the default?

Reply:
There is no default for this option in the action itself (see [here](https://github.com/codecov/codecov-action/blob/master/action.yml)).

Another thing is, apparently it didn't catch the latest failure, which is another error in the codecov-action action. I have reported it in the same issue as referenced below.